src/mappings/entities/mapIdentities.ts

```diff
@@ -50,6 +50,14 @@ export async function mapIdentities({
     if (eventId === EventIdEnum.SecondaryKeyPermissionsUpdated) {
       await handleSecondaryKeysPermissionsUpdated(blockId, params);
     }
+
+    if (eventId === EventIdEnum.PrimaryKeyUpdated) {
+      await handlePrimaryKeyUpdated(blockId, eventId, params);
+    }
+
+    if (eventId === EventIdEnum.SecondaryKeyLeftIdentity) {
```

Comment:
Sonar cloud complains because these are nested which doubles their complexity value.

If you switch the top conditional to be an early return, the complexity will drop well under the threshold.

I prefer to change the code since the smells might need to get reignored in the future, although this is OK to ignore if you prefer.